packages/apollo-link-http/README.md

```diff
@@ -0,0 +1,58 @@
+# HTTP Link
+
+## Purpose
+An Apollo Link to allow sending a single http request per operation.
+
+## Installation
+
+`npm install apollo-link-http --save`
+
+To use this package in a web browser or mobile app, you'll need a build system capable of loading NPM packages on the client.
+Some common choices include Browserify, Webpack, and Meteor +1.3.
+
+## Usage
+```js
+import HttpLink from "apollo-link-http";
+
+const link = new HttpLink({ uri: "/graphql" });
+```
+
+## Options
+HTTP Link takes an object with three options on it to customize the behavoir of the link. If your server supports it, the HTTP link can also send over metadata about the request in the extensions field. To enable this, pass includeExtensions as true
+
+|name|value|default|required|
+|---|---|---|---|
+|uri|string|"/graphql"|false|
+|includeExtensions|boolean|false|false|
+|fetch|ApolloFetch|ApolloFetch|false|
+
+By default, this link uses the [Apollo Fetch](https://github.com/apollographql/apollo-fetch) library for the HTTP transport.
+
+## Context
+The HTTP Link uses the `headers` field on the context to allow passing headers to the HTTP request.

```

Comment:
I think we need more options, specifically the `cors` stuff for `fetch`. AFAIK we should have some way to set arbitrary fetch options.

Reply:
ah true! I'll take a look. Honestly I think we could / should not use apollo-fetcher here since it duplicates a lot of stuff that could just be handled by a link. All of the `*wares* aren't needed with links. Thoughts?

Reply:
I don't have an opinion on this.

Reply:
The original reason I added apollo-fetch was to create an easy migration path from the current network interface. Not sure that is worthwhile anymore, since we want people entirely on Links.

Reply:
@evans totally! I think I'll adjust it and write an upgrade guide!